nucliadb/src/nucliadb/writer/api/v1/upload.py

```diff
@@ -638,6 +644,10 @@ async def upload_rid_prefix(
     x_password: Optional[list[str]] = Header(None),  # type: ignore
     x_language: Optional[list[str]] = Header(None),  # type: ignore
     x_md5: Optional[list[str]] = Header(None),  # type: ignore
+    x_processing_options: Optional[str] = Header(
+        default=None,
+        description="Base64 encoded FileProcessingOptions",
```

Comment:
Maybe you could say `FileProcessingOptions` serialized as JSON to be more explicit